resources/ts/Shared/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ComponentProps } from "react";
+import { classNames } from "../utils/classNames";
+
+type Props = ComponentProps<"div"> & {
+  visible: boolean;
+  text: string;
+};
+
+export const Tooltip = ({ visible, text, children }: Props) => {
+  return (
+    <>
+      <div className="w-[2.5rem]">
+        <div>
+          <div className="group relative inline-block">
+            {children}
+            <div
+              className={classNames(
+                "absolute left-full max-md:top-full top-2.5 max-md:left-1.5 z-20 ml-3 max-md:mt-3 -translate-y-1.5 max-md:-translate-x-1/2 whitespace-nowrap rounded bg-publiq-blue-dark py-[6px] px-4 text-sm font-semibold text-gray-100",
+                visible ? "visible" : "hidden"
+              )}
+            >
+              <span className="absolute left-[-3px] max-md:top-[1px] top-1/2 max-md:left-1/2 -z-10 h-2 w-2 -translate-y-1/2 max-md:-translate-x-1/2 rotate-45 rounded-sm bg-publiq-blue-dark"></span>
+              {text}
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
```

Comment:
```suggestion
      <div className="w-[2.5rem]">
        <div>
          <div className="group relative inline-block">
            {children}
            <div
              className={classNames(
                "absolute left-full max-md:top-full top-2.5 max-md:left-1.5 z-20 ml-3 max-md:mt-3 -translate-y-1.5 max-md:-translate-x-1/2 whitespace-nowrap rounded bg-publiq-blue-dark py-[6px] px-4 text-sm font-semibold text-gray-100",
                visible ? "visible" : "hidden"
              )}
            >
              <span className="absolute left-[-3px] max-md:top-[1px] top-1/2 max-md:left-1/2 -z-10 h-2 w-2 -translate-y-1/2 max-md:-translate-x-1/2 rotate-45 rounded-sm bg-publiq-blue-dark"></span>
              {text}
            </div>
          </div>
        </div>
      </div>
```

Fragments are not needed